sdf/1.7/schema/types.xsd

```diff
@@ -26,12 +26,13 @@
 
   <xsd:simpleType name="pose">
     <xsd:restriction base="xsd:string">
-      <xsd:pattern value="(\s*(-|\+)?(\d+(\.\d*)?|\.\d+|\d+\.\d+[eE][-\+]?[0-9]+)\s+){5}((-|\+)?(\d+(\.\d*)?|\.\d+|\d+\.\d+[eE][-\+]?[0-9]+))\s*"/>
+      <xsd:pattern value="(\s*(-|\+)?(\d+(\.\d*)?|\.\d+|\d+\.\d+)([eE][-\+]?[0-9]+)?\s+){5}(-|\+)?(\d+(\.\d*)?|\.\d+|\d+\.\d+)([eE][-\+]?[0-9]+)?\s*"/>
```

Comment:
We need to update this for 1.7 and up because `<pose/>` is now valid.

Reply:
Updated 1.7 to accept `<pose/>`: 75b33b6d6447182b5945832d8296cf883f55c908

Also updated 1.5 & 1.6 regex for `pose` (does not accept `<pose/>`) and `vector3d`